iset.mm

```diff
@@ -128314,7 +128314,7 @@ Norman Megill (2007) section 1.1.3.  Megill then states, "A number of
     "<IMG SRC='_wff.gif' WIDTH=24 HEIGHT=19 ALT=' wff' TITLE='wff'> ";
   althtmldef "wff" as
     '<SPAN CLASS=typecode STYLE="color:gray">wff </SPAN>'; /* was #00CC00 */
-  latexdef "wff" as "{\rm wff}";
+  latexdef "wff" as "{\mathrm{wff}}";
```

Comment:
I don't think we want the outer curly braces. At least we don't have them in existing entries like:

```
  latexdef "recs" as "\mathrm{recs}";
```